style.css

```diff
@@ -183,6 +187,144 @@ header {
   border: 0.5px solid #212121;
 }
 
+/* class = active  when see-project-button is clicked; active class is targeted */
+
+.active {
+  filter: blur(0.5rem);
+  pointer-events: none;
+  user-select: none;
+}
+
+/* Style Popup window */
+.modal {
+  position: fixed;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%) scale(1);
+  transition: 0.7s ease-in-out;
+  width: 90vw;
+  height: 90vh;
+  display: flex;
+  flex-direction: column;
+  z-index: 10;
+  border: 1px solid #dfe1e6;
+  border-radius: 10px;
+  background-color: #fff;
+  padding: 0 1rem 1rem 1rem;
+}
```

Comment:
- [ ] Kindly add a `overflow` scroll the mobile menu :+1: .The content overflows make sure you solve that :smile: 

[Screencast from 02-03-2023 11_41_36 AM.webm](https://user-images.githubusercontent.com/80458199/216573626-55ab11a4-f0cd-4afb-833c-5d2affed439b.webm)

Reply:
Thank you @iambenkis for pointing this out. We have added the overflow-y: scroll to the modal in order to fix the issue.

.modal {
overflow-y: scroll;
}